### DISCUSSION TITLE
永远喜欢甘雨

### DISCUSSION THREAD

**Discussion by tianxianuo** [share121/inter-knot#1694] [其他]
![17202603778016c8d9e339bdbdbf6879495aaaae256d201e923c2b3de8e0f240548d1369f3465 0](https://github.com/user-attachments/assets/0b94257e-d807-4a1f-807b-870e58853caa)
永远喜欢甘雨
甘雨最可爱✧٩(ˊωˋ*)و✧

**Comment by tianxianuo** [comment]
_(:зゝ∠)_

**Comment by nahida0-o** [comment]
她真好看

**Comment by Wangsheng-Zhongli** [comment]
甘雨是璃月港的忠诚守护者，她的温柔与坚韧早已在这片土地上深深扎根。你对她的喜爱，亦是对她的尊重与欣赏。然，甘雨肩负重责，独守孤高，希望你能在心中为她留一份敬重，不仅喜其美，亦念其心。
![Uploading 钟离喝茶.png…]()

**Comment by AnbulesPeo** [comment]
她真好看

**Comment by zhang090613** [comment]
好好康啊

**Comment by duskbirds** [comment]
英雄所见略同<img src="https://bbs-static.miyoushe.com/static/2023/04/28/ad3e4d3d37302d240339f611b2458ef9_7921317972787298019.png" alt="米仔_比心" width="75" >

**Comment by real-strange** [comment]
终于看到稀有的甘雨÷了
